third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -529,8 +538,89 @@ impl InterpreterImpl {
                         loader,
                         function,
                         frame_cache,
+                        ClosureMask::empty(),
+                        vec![],
                     )?;
                 },
+                ExitCode::CallClosure(_sig_idx) => {
+                    // TODO(#15664): runtime check whether closure value on stack is same as sig_idx
+                    let (fun, captured) = self
+                        .operand_stack
+                        .pop_as::<Closure>()
+                        .map_err(|e| set_err_info!(current_frame, e))?
+                        .unpack();
+                    let lazy_function = LazyLoadedFunction::expect_this_impl(fun.as_ref())
+                        .map_err(|e| set_err_info!(current_frame, e))?;
+                    let mask = lazy_function.closure_mask();
+                    let function = lazy_function
```

Comment:
Can you add a comment here saying that if the function is not present, loading would happen here.